### PR TITLE
Change uint64 to bytes for supporting big number

### PR DESCRIPTION
**Pull request by kjunu**
https://github.com/aergoio/aergo/issues/16

### CODE REVIEW

File: account/accountservice_test.go [hunk 1]
```diff
@@ -155,7 +156,7 @@ func TestVerfiyFail(t *testing.T) {
 	assert.NotNil(t, tx.Body.Sign, "failed to sign")
 
 	//edit tx after sign
-	tx.Body.Amount = 0xff
+	tx.Body.Amount = new(big.Int).SetUint64(0xff).Bytes()
```
Comment (graup) [inline account/accountservice_test.go]: Isn't `big.NewInt(0xff).Bytes()` the same as `new(big.Int).SetUint64(0xff).Bytes()`?

File: cmd/aergocli/cmd/staking.go [hunk 2]
```diff
@@ -16,55 +18,56 @@ import (
...
+		return errors.New("Failed: minimum staking value is " + types.StakingMinimum.String())
 	}
 
 	tx := &types.Tx{
 		Body: &types.TxBody{
 			Account:   account,
 			Recipient: []byte(types.AergoSystem),
-			Amount:    amount,
-			Price:     0,
+			Amount:    amountBigInt.Bytes(),
```
Comment (graup) [inline cmd/aergocli/cmd/staking.go]: Just a question, what is the point of calling Bytes()? Is it to create a copy?

File: account/accountservice_test.go (hunk 1 above)
Comment (kjunu) [reply account/accountservice_test.go]: right 👍 I will fix :)

File: cmd/aergocli/cmd/staking.go (hunk 2 above)
Comment (kjunu) [reply cmd/aergocli/cmd/staking.go]: yes it's copy :(

https://github.com/golang/go/blob/master/src/math/big/int.go#L440

**Pull request merged**